packages/flutter/test/gestures/drag_test.dart

```diff
@@ -553,9 +554,9 @@ void main() {
 
   testGesture('Drag with multiple pointers in down behavior - default', (GestureTester tester) {
     final HorizontalDragGestureRecognizer drag1 =
-    HorizontalDragGestureRecognizer() ..dragStartBehavior = DragStartBehavior.down;
+      HorizontalDragGestureRecognizer() ..dragStartBehavior = DragStartBehavior.down;
```

Comment:
What's with the indentations here and below?

Reply:
Shouldn't it be indented here?